client/src/components/InfoBar.vue

```diff
@@ -10,6 +10,7 @@
           class="w-4 h-4 text-blue-500"
           v-if="index != path.length - 1"
         />
+        <!-- TODO: show different Icon for json -->
```

Comment:
Ist das TODO nicht abgeschlossen?

Reply:
Ne unten in der Info Bar wird noch das "PhotographIcon" angezeigt. Für json files finde ich das nicht so passend (ist ja kein Bild), da will ich noch ein anderes raussuchen.